Workbooks/UpdateCompliance/UpdateComplianceHub/Update Compliance Hub.workbook

```diff
@@ -0,0 +1,1660 @@
+                    "crossComponentResources": [
+                      "{mappedWorkspace}"
+                    ],
+                    "visualization": "piechart",
+                    "chartSettings": {
+                      "yAxis": [
+                        "SecurityUpdateStatus"
+                      ],
+                      "seriesLabelSettings": [
+                        {
+                          "seriesName": "Latest",
+                          "color": "blue"
+                        },
+                        {
+                          "seriesName": "NotLatest",
+                          "label": "Not Latest",
+                          "color": "orange"
+                        },
+                        {
+                          "seriesName": "MultipleSecurityUpdatesMissing",
+                          "label": "Multiple Security Updates Missing",
+                          "color": "red"
+                        }
+                      ]
+                    }
+                  },
+                  "name": "OverallSecurityUpdateStatusChart"
+                },
+                {
+                  "type": 11,
+                  "content": {
+                    "version": "LinkItem/1.0",
+                    "style": "paragraph",
+                    "links": [
+                      {
+                        "id": "f3d59e76-ffb9-42a8-b912-b034e54167a4",
+                        "linkTarget": "WorkbookTemplate",
+                        "linkLabel": "View more details in quality updates",
+                        "style": "link",
+                        "workbookContext": {
+                          "componentIdSource": "workbook",
+                          "resourceIdsSource": "workbook",
+                          "templateIdSource": "static",
+                          "templateId": "community-workbooks/UpdateCompliance/Quality Updates",
+                          "typeSource": "workbook",
+                          "gallerySource": "workbook",
+                          "locationSource": "default"
+                        }
+                      }
+                    ]
+                  },
+                  "name": "GoToQUChartLink",
+                  "styleSettings": {
+                    "margin": "0px 0px 0px 12px"
+                  }
+                }
+              ],
+              "exportParameters": true
+            },
+            "conditionalVisibility": {
+              "parameterName": "currentWorkbook",
+              "comparison": "isEqualTo",
+              "value": "Summary"
+            },
+            "name": "SummaryContentGroup",
+            "styleSettings": {
+              "margin": "0",
+              "padding": "0"
+            }
+          },
+          {
+            "type": 12,
+            "content": {
+              "version": "NotebookGroup/1.0",
+              "groupType": "editable",
+              "loadType": "always",
```

Comment:
Is this a group that's supposed to load another workbook? there's a way to just reference that other work book

![image](https://user-images.githubusercontent.com/7664931/176317320-211b7689-7c3b-4706-80cc-506e20a0f1f6.png)

That way you don't need to store all the content of the other workbooks. Follow this pattern if one workbook loads another. This is just the first one that caught my eye.

Also I would leave the loadType to `lazy` which only loads when necessary (visible) otherwise does not load the workbook if hidden

Reply:
I actually made this one editable because it uses a variable from the main Hub workbook. Since it's a separate workbook that is just being loaded into this group I didn't know how to reference that variable otherwise. I attached a screenshot below; the variable being used is "workspace" which only exists in the Hub workbook. The Quality Updates workbook needs it to call queries.

![image](https://user-images.githubusercontent.com/98173238/176494960-7949259d-a1a5-4396-b36b-a974c1d7959b.png)

Reply:
The Quality Updates workbook can havea a workspace parameter as a place holder. When the hub workbook loads the quality updates workbook, Quality updates workbook will inherit the value from the Hub workbook